### ISSUE TITLE
[ament_cpplint] does not detect explicit keyword when on another line

### ISSUE THREAD

**Issue by wjwwood** [ament/ament_lint#310]
Sometimes I get `Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]` when I have code like this:

```c++
class Foo
{
public:
  explicit
  Foo(int arg) {}
};
```

The constructor is obviously marked `explicit`, but `cpplint` doesn't detect it properly.

**Comment by clalancette** [comment]
I just tested this out with the newer cpplint, and it seems to handle this situation now.  So I'm going to close this out, but feel free to reopen if I'm mistaken.

**Comment by clalancette** [comment]
Ah, never mind.  It's still a problem in more complicated situations, reopening.
